pkg/apis/ark/v1/config.go

```diff
@@ -98,6 +98,7 @@ type AWSConfig struct {
 	DisableSSL       bool   `json:"disableSSL"`
 	S3ForcePathStyle bool   `json:"s3ForcePathStyle"`
 	S3Url            string `json:"s3Url"`
+	KMSKeyID         string `json:"kmsKeyId"`
```

Comment:
It's possible some of the discrepancies are from me manually generating some tags vs using vim-go's `:GoAddTags` - can't say for sure. Consistency would be nice... But not something we need to change right now. Especially since it breaks backwards compat.